src/Modules/CalcOffence.lua

```diff
@@ -2961,6 +2968,9 @@ function calcs.offence(env, actor, activeSkill)
 				local effectMod = calcLib.mod(skillModList, dotCfg, "AilmentEffect")
 				output.BaseBleedDPS = baseVal * effectMod * rateMod * effMult
 				bleedStacks = m_min(maxStacks, (output.HitChance / 100) * globalOutput.BleedDuration / output.Time)
+				if skillFlags.totem then
```

Comment:
This `if` is not really necessary given that `local activeTotems` will be 1 when no totems, or a valid value if totems.

There is an extra space near the multiplication of `output.Time  * activeTotems`